templates/quickstart-crowd-dc-with-vpc.template.yaml

```diff
@@ -224,7 +224,7 @@ Parameters:
     AllowedValues: ["Off", "Metrics Only", "Metrics and Logs"]
     ConstraintDescription: "Must be 'Off', 'Metrics Only', or 'Metrics and Logs'"
   ClusterNodeInstanceType:
-    Default: c5.xlarge
+    Default: t3.medium
```

Comment:
Why the t family of instance classes?

Reply:
So I decided to use that as the default value because that's what's used for the [Crowd quickstart](https://bitbucket.org/atlassian/atlassian-aws-deployment/src/master/quickstarts/quickstart-crowd-master.template.yaml) located under the BitBucket repo. Looking at the default's used by the official quickstarts for `Jira`, `Confluence` and `BitBucket` they all use the `c` family i.e. `c5.xlarge` which makes sense for the workload. I'll update this.

Reply:
I am not sure what is recommended instance size for Crowd. Values for other products were done based on performance tests for their specific use case, for example:
https://confluence.atlassian.com/enterprise/infrastructure-recommendations-for-enterprise-jira-instances-on-aws-969532459.html

For Crowd, I've found only https://confluence.atlassian.com/enterprise/crowd-data-center-performance-935910743.html but they use `m4` type. Would be good to check with Crowd team.

Reply:
Will do.